components/RestaurantCard.tsx

```diff
@@ -53,6 +55,9 @@ const RestaurantCard = ({ restaurant }: any) => {
     <div
       key={restaurant.name}
       className="border-[1px] w-full h-full border-gray-200 mb-2 group shadow-md rounded-xl"
+      onClick={() => {
+        router.push(`restaurants/${restaurant.name}`)
+      }}
```

Comment:
It wasn't obvious that the user needs to click the `+ menu` button, so this navigates to the restaurant page if they click anywhere on the preview tile

Reply:
Good idea - I like the feedback on UI stuff like this